### ISSUE TITLE
how to install rtklib on Raspberry pi or Linux platform? 

### ISSUE THREAD

**Issue by atanusan** [tomojitakasu/RTKLIB#281]
I am new in rtklib.
I want to install rtklib on Raspberry pi or Linux. Please suggest me with your kind information. 
Thank you 
Best regards.

**Comment by tomojitakasu** [comment]
Read 4.2 of the manual.
More detailed instructions will be added in the future release.

**Comment by atanusan** [comment]
I have connected raspberry pi (ubuntu mate) with javad receiver by usb to rs232 converter cable. When I ran the command ./rtkrcv -s, I found '"rtk server start error". Is this USB related problem? 
Please suggest me.
Best regards. 
Atanu Santra

**Comment by alexmyczko** [comment]
If it is .deb based you can use the source package here: http://sid.ethz.ch/debian/rtklib/upload/

**Comment by atanusan** [comment]
Dear alexmyczko,
Thanks for the information.
Best regards.

**Comment by Wilkins089** [comment]
I have a question, it is possible to put it to run on linux, what would be the complications of it, thanks.

**Comment by joriatyBen** [comment]
Maybe you would read the rtklib manual first and try it by yourself. When
you will come up with a specific question it would be easier to help you.

Wilkins Ortiz <notifications@github.com> schrieb am Mo., 20. Aug. 2018,
19:20:

**Comment by hadisehamiri** [comment]
hi, I have a problem running rtklib on linux. I would be grateful if anybody help me. I gathered some data with a pair of ublox_m8t receivers and saved tag files. I re ran those data with rtknavi on windows and raspberry board successfully but I failed to run it on a virtual machine. It seems that the virtual machine doesn't consider .tag files. these ore linux versions installed on raspberry board and virtual machine:
raspberry board:
Distributer ID: Raspbian
Description: Raspbian GNU/Linux 8.0 (jessie)
code name: jessie
virtual machine:
Distributer ID: ubuntu
Description: ubuntu 16.10
code name: yakkety

the version of rtklib is demo5 b31 from rtkexplorer website.
thanks
@tomojitakasu
